src/odemis/acq/stream/_projection.py

```diff
@@ -1191,9 +1191,9 @@ def projectAsRaw(self):
         try:
             data = self.stream.calibrated.value
             raw_md = self.stream.calibrated.value.metadata
-            md = {k: raw_md[k] for k in (model.MD_PIXEL_SIZE, model.MD_POS) if k in raw_md}
+            md = {k: raw_md[k] for k in (model.MD_PIXEL_SIZE, model.MD_POS, model.MD_THETA_LIST) if k in raw_md}
 
-            # Average time values if they exist.
+            # Average time/theta values if they exist.
```

Comment:
time devided by theta, time or theta, or time and theta? I think writing this out might help someone scanning the code (like me) to understand what is happening. The "/" might confuse a bit, though reading the code could probably give me the answer.